src/NFun.Fuspectests/FuspecTestParserTests.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using FuspecTests;
+using Nfun.Fuspec.Parser;
+using Nfun.Fuspec.Parser.Model;
+using NFun.Fuspec.TestHandler;
+using NFun.Fuspec.TestHandler.Models;
+using NUnit.Framework;
+
+namespace NFun.Fuspectests
+{
+    public class FuspecTestParserTests
+    {
+        [Test]
+        public void FuspecParser_ParseAndRunFuspec_ReturnParsedTests()
+        {
+            //todo пример использования fuspecParser
+            FuspecTestCase[] testCases = GenerateFuspecTestCases(FuspecTestCasesCollection.TestCollection);
+            IEnumerable<BaseFuspecTestCaseResult> testsResult = FuspecTestBase.RunTests(testCases);
+
+            Assert.Pass();
+        }
+
+
+
+
+
+
+        
+
+        private static Stream GenerateStreamFromString(string s)
```

Comment:
This stack overflow answer is not the best ;)

the best is https://stackoverflow.com/a/38399723/1475183

To use string extensions is much readeable and intuitive

Reply:
@tmteam it's not mine:)) I pulled Natasha's code out of the tests and separated it with special spaces to fix it later